notebooks/GWAS_1000_genomes_p2/requirements.txt

```diff
@@ -0,0 +1,6 @@
+terra-notebook-utils
+herzog >= 0.0.2, < 0.1.0
+pandas
+seaborn
+hail
+tenacity
```

Comment:
I would pin these.  You can see your current env and the versions with `pip freeze`.  If you pin the versions you see there, it should be a working env.  same for the others.

Reply:
Addressed in [this commit](https://github.com/DataBiosphere/bdcat_notebooks/commit/5f1e293fb93dfedd7d6fb8cb14b7b94f57489072) on new branch.

Reply:
Addressed [here](https://github.com/DataBiosphere/bdcat_notebooks/commit/bbc54853bc30be6fff14222daac52019e0b4e0ed) (and in a few other commits)